docs/reference/templates.rst

```diff
@@ -202,19 +191,17 @@ can specify the templates to use in the ``Admin`` service definition:
 
 Changes made using the ``setTemplate()`` and ``setTemplates()`` methods
 override the customizations made in the configuration file, so you can specify
-a global custom template and then override that customization on a specific
+a global custom template and then override that customization for a specific
 ``Admin`` class.
 
 Finding configured templates
 ----------------------------
 Each ``Admin`` has a ``TemplateRegistry`` service connected to it that holds
-the templates registered through the configuration above. Through the method
-``getTemplate($name)`` of that class, you can access the templates set for
-that ``Admin``. The ``TemplateRegistry`` is available through ``$this->getTemplateRegistry()``
-within the ``Admin``. Using the service container the template registries can
-be accessed outside an ``Admin``. Use the ``Admin`` code + ``.template_registry``
-as the service ID (i.e. "app.admin.post" uses the Template Registry
-"app.admin.post.template_registry").
+the templates registered through the configuration above. The ``TemplateRegistry``
+is available through ``$this->getTemplateRegistry()`` within the ``Admin``.
+Using the service container the template registries can be accessed outside an ``Admin``.
```

Comment:
That sentence lacks punctuation IMO, also it might be better not to mention the service container other one people might end up using `$container->get` over DI.
```suggestion
Outside an ``Admin``, you can obtain the template registries with their service ID.
```